modules/ld/src/main/scala/ch/epfl/bluebrain/nexus/admin/ld/jena/JenaJsonLD.scala

```diff
@@ -32,10 +32,11 @@ private[ld] final case class JenaJsonLD(json: Json, private[jena] val modelF: ()
     val (subs, objs) = graph
       .find(Node.ANY, Node.ANY, Node.ANY)
       .asScala
-      .withFilter(triple => triple.getSubject.isBlankOrUri && triple.getObject.isBlankOrUri)
+      .withFilter(triple => triple.getSubject.isBlankOrUri)
```

Comment:
why that change?

Reply:
`triple.getSubject.isBlankOrUri && triple.getObject.isBlankOrUri` would select nothing for a payload that only has literal objects in the RDF triples.